libGitWrap/Private/DiffListPrivate.hpp

```diff
@@ -36,6 +36,7 @@ namespace Git
         {
         public:
             explicit DiffListPrivate(const RepositoryPrivate::Ptr& repo, git_diff* diff);
+            explicit DiffListPrivate(RepositoryPrivate* repo, git_diff* diff);
```

Comment:
Thanks! I wanted to ask either, if that was a wise decision - naughty me :smile:. I'll fix it before merge.